src/interface/variable_pack.hpp

```diff
@@ -80,9 +80,13 @@ class VariablePack {
     return v_(n)(k, j, i);
   }
   KOKKOS_FORCEINLINE_FUNCTION
+  T &operator()(const int m, const int n, const int k, const int j, const int i) const {
+    return v_(n)(k, j, i);
```

Comment:
That may introduce its own new code that needs to be maintained.

Reply:
Yes, of course. But if the new code is only 10% of the old code and logically simpler and/or more flexible then this would be a change worth pursing.

Now that I think about it, what's the status down stream adoption/testing of MeshBlockPacks for real ParArrayND data?
I'm asking as my current impression is that the design of the MeshBlockPacks silently assumed that `(b[block], v[ariable] , k, j, i)` indexing is used.
As far as I

Reply:
@jdolence has been working on an adapting to meshblock packs in a branch but it's not in our master branch yet. Yes, I think we assume that block,variable,k,j,i indexing everywhere.

Reply:
> Yes, of course. But if the new code is only 10% of the old code and logically simpler and/or more flexible then this would be a change worth pursing.

I'm not convinced that would be the case... We use `ParArrayND`s fairly extensively in our downstream code for, e.g., vector-valued variables.